src/lib/git.ts

```diff
@@ -225,7 +225,7 @@ export async function checkSyncStatus(git: Simple): Promise<SyncStatus> {
     : 'diverged'
 }
 
-export interface BasicGithubRepoInfo {
+export interface BasicGitHubRepoInfo {
```

Comment:
I'm not sure how public your API is.

It's fairly trivial for me to split out changes like this, or to drop them entirely.

Mostly, the goal is to fix spelling of the brand when written standalone as `GitHub`. Just as one shouldn't write `priSMa laBS`.

Reply:
This fine fine!